ddl/db_test.go

```diff
@@ -1711,6 +1711,55 @@ func (s *testDBSuite) TestCreateTableWithPartition(c *C) {
 		  partition p3 values less than (18446744073709551000 + 1),
 		  partition p4 values less than (18446744073709551000 + 10)
 		);`)
+
+	// Only range type partition are now supported, range columns partition only implement parser so do not check.
```

Comment:
Only range type partition **is** now **supported. Range** columns partition only **implements the** **parser, so it will not be checked.**